content/en/users/tutorials/accessing-vm-with-ssh/_index.md

```diff
@@ -0,0 +1,220 @@
+---
+title: "Accessing virtual machines with SSH"
+linkTitle: "Accessing VMs with SSH"
+type: docs
+weight: 20
+description: "Accessing virtual machines with SSH"
+---
+
+## Overview
+
+This page provides an introduction of connecting from a local computer to a
+cloud host via SSH. It provides general guidelines, SSH options, tips, and
+examples for setting up the OpenStack environment.
+
+## Username and password
```

Comment:
I think this should not be the first option, but rather be put after SSH keys and clearly state that this is not to be used unless there is a very strong need (?)

Reply:
I've tried to address this, please check.